lib/gollum-lib/page.rb

```diff
@@ -101,7 +101,7 @@ def name
     #
     # Returns the fully sanitized String title.
     def title
-      Sanitize.clean(name).strip
+      @wiki.sanitizer.clean(name).strip
```

Comment:
Alternatively we could make a direct Loofah `:strip` or `:prune` call here (no need for the postprocessing in the `Sanitizer` class here).